12_listcomprehension_lab/02.py

```diff
@@ -1,6 +1,11 @@
-"""
-Write a python program that takes two words
-as sys.argv and prints only the letters
-common to both
-"""
-
+
+"""
+This program prints all possible permutation of 3 small letters
+"""
+
+import sys
+
+letters = [chr(i) for i in range(97,123)]

```

Comment:
Python allows any number of lists in the list comprehension, so no need for two lines:

```
print [a + b + c for a in letters for b in letters for c in letters]
```